docs/source/api/bentoml.rst

```diff
@@ -62,10 +62,6 @@ ver
 +++
 .. autofunction:: bentoml.ver
 
-save
```

Comment:
let's keep this?  I don't think it's necessary to introduce this breaking change here.